pkg/operator/bootstrap.go

```diff
@@ -129,6 +132,10 @@ func RenderBootstrap(
 		bundle = append(bundle, filesData[kubeAPIServerServingCA]...)
 		spec.KubeAPIServerServingCAData = filesData[kubeAPIServerServingCA]
 	}
+	// Set the cloud-provider CA if given.
+	if _, ok := filesData[cloudProviderCAFile]; ok {
+		spec.CloudProviderCAData = filesData[cloudProviderCAFile]
```

Comment:
you're also matching an existing pattern here, but I would have expected:

```go
if data, ok := filesData[cloudProviderCAFile]; ok {
  spec.CloudProviderCAData = data
}
```

I'm fine with any of "we're just pattern matching, this is fine", "we'll diverge from the pattern to avoid a doubled lookup (which maybe gets optimized out in compilation and won't take long anyway", or "nice.  And we'll fix `kubeAPIServerServingCA` above as well".  And even if I'm not fine with something, I'm not a maintainer here, so don't worry about it ;)

Reply:
I like that